src/julia/install.sh

```diff
@@ -57,4 +57,11 @@ su "${USERNAME}" -c "curl -fsSL https://install.julialang.org | sh -s -- --yes -
 # Clean up
 cleanup_apt
 
+# Copy postcreate.jl into the container
+echo "Copying postcreate.jl into the container..."
+SCRIPT_DIR=/usr/local/julia-devcontainer-features/
+mkdir -p "${SCRIPT_DIR}"
+cp postcreate.jl "${SCRIPT_DIR}/"
+chmod +x "${SCRIPT_DIR}/postcreate.jl"
```

Comment:
I think it makes more sense to make the script included in the Feature executable than to do this here.

Reply:
I took that from https://github.com/devcontainers/features/blob/a9a7a25d60585ce2145374ec6825cbdc7f16cb90/src/common-utils/main.sh#L488. At least when trying out the feature from my local filesystem, looks like the `chmod +x` is not necessary even if the source file was not executable (I agree though that it should be executable).

I'll make `postcreate.jl` executable in the repo, and drop the `chmod` at line 65.